### PR TITLE
Rewrite of HA section, hopefully correct this time

### PR DESCRIPTION
**Pull request by cjschroder**

### CODE REVIEW

File: xml/cap_depl_install_production.xml [hunk 1]
```diff
@@ -915,51 +917,80 @@ suse/uaa      2.8.0    A Helm chart for SUSE UAA
...
   <para>
 This example &ha; configuration needs two separate configuration files, one for 
-UAA and one for SCF. The first example is for UAA, 
-<filename>uaa-sizing.yaml</filename>.
+UAA and one for SCF. It may be helpful to read the comments in the 
+<literal>sizing:</literal> section in the default
+<filename>values.yaml</filename> file for each namespace, which describes which
+roles can be scaled and the scaling options for each role. One way is to use 
+<command>helm inspect</command> to read the <literal>sizing:</literal> section 
+in your installed charts:
```
Comment (mook-as) [inline xml/cap_depl_install_production.xml]: The charts don't need to be _installed_ (as in, having run `helm install suse/cf`); they just need to be in a listed repo.

File: xml/cap_depl_install_production.xml [hunk 2]
```diff
@@ -915,51 +917,80 @@ suse/uaa      2.8.0    A Helm chart for SUSE UAA
...
+unpack them, and then use Perl to extract the information for each role from the 
+<literal>sizing:</literal> section. The following example is for UAA.
+You may see a few non-relevant lines from the sections preceding and following 
+the <literal>sizing:</literal> section. In the following example these are lines 
+29-53, and lines 226-245.
 </para>
+
+<screen>
+&prompt.user;helm fetch suse/uaa
+&prompt.user;tar xzvf cf-2.8.0.tgz
```
Comment (mook-as) [inline xml/cap_depl_install_production.xml]: How did you fetch _uaa_ and extract _cf_? Shouldn't those match?

File: xml/cap_depl_install_production.xml [hunk 3]
```diff
@@ -915,51 +917,80 @@ suse/uaa      2.8.0    A Helm chart for SUSE UAA
...
-  etcd:
-    count: 3
-  loggregator:
-    count: 2
   mysql:
-    count: 1
-  nats:
+    count: 3
+  mysql_proxy:
     count: 2
```
Comment (mook-as) [inline xml/cap_depl_install_production.xml]: I don't think we supported (correctly) scaling `mysql-proxy` yet.  The as-shipped values file mistakenly thought we could, though, I believe.

File: xml/cap_depl_install_production.xml [hunk 4]
```diff
@@ -968,37 +999,54 @@ sizing:
...
     count: 3
+  doppler:
+    count: 4
   loggregator:
-    count: 2
+    count: 7
   mysql:
     count: 3
+  mysql_proxy:
+    count: 3
```
Comment (mook-as) [inline xml/cap_depl_install_production.xml]: Same as above; don't scale this yet.

File: xml/cap_depl_install_production.xml (hunk 2 above)
Comment (cjschroder) [reply xml/cap_depl_install_production.xml]: Magic!

File: xml/cap_depl_install_production.xml (hunk 4 above)
Comment (cjschroder) [reply xml/cap_depl_install_production.xml]: Yikes. The whole point of this section is showing customers how to get correct information. I shall note these exceptions.

**Pull request merged**